CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 5.1.0 (in progress)
 ================
+* [#1222](https://github.com/oshi/oshi/pull/1222): Added messages to unit test assertions in GraphicsCardTest.java. - [@david145noone](https://github.com/david145noone).
 * Your contribution here
-
```

Comment:
One last change: you need to leave a space before the next header, otherwise this shows as "Your contribution here 5.0.0 (5/5/2020), ..."